### DISCUSSION TITLE
How to disable sending metrics?

### DISCUSSION THREAD

**Discussion by mmuenker** [caprover/caprover#1357] [General]
I have installed and setup caprover.

Now I see that the container send request to `http://opendata.baidu.com/api.php` with my in PLAIN TEXT.

The IP address send in the query is not obfuscated and the connection is not over https.
This is not acceptable!!!!

**Comment by githubsaturn** [comment]
Is this from the caprover container? CapRover doesn’t have any metrics yet. You are welcome to study the code.

**Reply by mmuenker** [reply]
yes i found this in the logs from the `caprover/caprover:1.10.1` container.

```
GET http://opendata.baidu.com/api.php?query=189.xxx.xxx.186&co=&resource_id=xx027&oe=utf8
```

and another request to `news.sina.com.cn`

```
GET http://news.sina.com.cn/robots.txt
```

I am currently reviewing the code.

**Reply by githubsaturn** [reply]
1- Where did you see these logs? Using `docker service logs`? 
2- Are you using the official image or a mirrored one?

**Reply by mmuenker** [reply]
1. with `docker logs captain-captain.1.5xxxxxx`
2. official

REPOSITORY | TAG | IMAGE ID | CREATED
--- | --- | --- | ---
caprover/caprover | 1.10.1 | 8729a3006161 | 4 months ago

**Comment by githubsaturn** [comment]
Can you please copy the snippet of the logs that you are referring to? Does it happen when you can certain actions or it just randomly appears from time to time?

PS: I suspected that it may be coming from an npm dependency, but I `grep`ed the files and couldn't find anything either.
